collect_app/src/test/java/org/odk/collect/android/injection/TestModule.java

```diff
@@ -63,18 +59,4 @@ RxEventBus provideRxEventBus() {
         return new RxEventBus();
     }
 
-    @PerApplication
-    @Provides
```

Comment:
@xsteelej why were these removed if they are now being used inside `HttpClientConnection`? If it were due to dagger issue then with the whole approach of adding all the dependencies in one Component/Module then you can inject these here! That will make it easier for us to make more indepth tests by providing mocked versions via DI if necessary.

Reply:
Hi @jd-alexander, I removed them because they are specific to httpclient and probably won't be needed for another HttpInterface such as okhttp3.